### ISSUE TITLE
failed fetching new block template from kaspa: route 'rpc client - incoming' got timed out 

### ISSUE THREAD

**Issue by zro-1** [kaspanet/kaspad#2293]
so i cant seem to rpc to my kaspad running the node from a device in my local network and i have followed and read the instructions on every doc i can find im running windows so i know thats my first problem but any help would be great ty. Oh to reproduce using the kdx wallet desktop kaspad and ks_bridge. i can get my pc gpu to mine and i can see out side connections join but never any shares submit other then my gpu.

**Comment by Jomossey** [comment]
@zro-1 sounds like you're having a bit of a struggle with your kaspad setup on Windows. It's like trying to teach a pigeon to sing like a canary, right?

**Comment by zro-1** [comment]
I 've been able to launch and mine through a VM hosted on another VM decently well, but it's not what I was hoping to achieve. So, I'll continue to read all these files; maybe they will make sense or 'cents' by the time I figure it out. I hope to join the testnet new method, as I’m building an old server and waiting on a new server to be shipped.

But I feel I should bring up a security concern: I found someone trying to harm my computer with matching end locations via nmap. I was DDoSed multiple times every time I launched, and they sent my wife harassing emails to both our accounts. I tracked it as well as I could, but I had to reset my PC, something I haven't had to do for years, even before getting a CIT degree.

I'm not sure if it's files from GitHub or the opening of the stratum port (16110 or 16111), but I had saferpc set to 127.0.0.1 for most cases, and still, my system got wrecked. Excuse the language, but I hope that these weren't intentionally placed by developers to test people's patience or to indulge their inner troll.
